demos/kubernetes-helm/README.md

```diff
@@ -8,17 +8,17 @@ The latest stable helm chart can be found [here](https://github.com/helm/charts/
 Now grab a copy of the helm chart [values file](https://github.com/helm/charts/blob/master/stable/jenkins/values.yaml) and adjust the Master part a little bit:
 
 ```yaml
-Master:
-  Name: jenkins-master
-  Image: "jenkins/jenkins"
-  ImageTag: "2.134-alpine"
-  ImagePullPolicy: "IfNotPresent"
-  Component: "jenkins-master"
+master:
+  componentName: jenkins-master
+  Image: 'jenkins/jenkins'
```

Comment:
looking at the [docs](https://github.com/helm/charts/tree/master/stable/jenkins) the rest of fields should be updated as well? they are no longer upper case

Reply:
truth, I will adjust, thanks for the warning